tests/unittests/utils3d_tests.py

```diff
@@ -0,0 +1,282 @@
+"""
+FiftyOne 3D utilities unit tests.
+
+| Copyright 2017-2023, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import os
+import tempfile
+import unittest
+
+import numpy as np
+import open3d as o3d
+from PIL import Image
+from typing_extensions import Literal
+
+import fiftyone as fo
+import fiftyone.utils.utils3d as fou3d
+from decorators import drop_datasets
+
+
+class BaseOrthographicProjectionTests(unittest.TestCase):
+    def setUp(self) -> None:
+        self.temp_dir = tempfile.TemporaryDirectory()
+        self.test_pcd_path = os.path.join(self.temp_dir.name, "test_pcd.pcd")
+
+    def tearDown(self) -> None:
+        self.temp_dir.cleanup()
+
+    def assertValidProjection(self, metadata_field, expected_image_path=None):
```

Comment:
Some of these tests fail locally for me with the following error:

```
FileNotFoundError: [Errno 2] No such file or directory: 'specs/3d/100x100_seed_1_none.png'
```

Seems like an absolute vs relative path issue?